### PR TITLE
Telegram: Initialize device_groups.md

### PR DESCRIPTION
**Pull request by AnierinBliss**
Signed-off-by: AnierinB <anierin@evolution-x.org>

### CODE REVIEW

File: telegram/device_groups.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+- [Xiaomi Mi 9T Pro & Redmi k20 Pro](https://t.me/EvolutionXRaphael)
+
+- [Xiaomi Mi 10T & 10 Pro](https://t.me/EvolutionXApollo)
+
+- [Xiaomi Mi Mix 2S](https://t.me/EvolutionXPolaris)
+
+- [Xiaomi Pocophone F1](https://t.me/EvolutionXBeryllium)
+
+- [Xiaomi Poco F2 Pro](https://t.me/EvolutionXKona)
```
Comment (RealAkito) [inline telegram/device_groups.md:60]: Is actually Xiaomi Gourami (kona)

**Review by RealAkito — CHANGES_REQUESTED** [review]

File: telegram/device_groups.md (hunk 1 above)
Comment (AnierinBliss) [reply telegram/device_groups.md]: Fixed

**Review by RealAkito — APPROVED** [review]

**Comment by RealAkito** [comment]
Why are we initializing a new folder for just a MD file?

**Comment by AnierinBliss** [comment]
> Why are we initializing a new folder for just a MD file?

I mean we can put it in the xda folder or in the root of this tree. Open to suggestions

**Comment by RealAkito** [comment]
Hmm, If you could change that it'd be great.

**Pull request merged**